pyhap/accessory.py

```diff
@@ -18,8 +21,8 @@
 class Category:
```

Comment:
I thought you wanted to change this to dict?

Reply:
I would have loved to. However I don't think that it quite works here. At the moment this version seems to be the best.

For the future we could think about what is stored in the `accessory`. As far as I can tell, it's the category number at the moment. With a change to a dictionary it might make sense to store the constant name and assign the number later when sending the acc infos to HomeKit.
I didn't wanted to have this discussion here which is the reason I changed it back, so this could get merged.

Reply:
What is the purpose of having it like e.g. string in the Acc? Readability?

Reply:
Using a dictionary might pose problems when importing a category.
...

While I was thinking about it, are their problems with just doing the following?
```py
CAT_OTHER = 1
CAT_BRIDGE = 2
# ...
```
That would solve the import problem and doesn't need a dict.

---

For comparison the dict variant I thought of was:
```py
CAT_OTHER = 'other'
CAT_BRIDGE = 'bridge'

CATEGORIES = {
    CAT_OTHER: 1,
    CAT_BRIDGE: 2,
}
```

Reply:
No problems, I would just name it CATEGORY_OTHER, etc.

Reply:
Would you mind if I move them to the `const` file?

Reply:
Go ahead